lib/geo.js

```diff
@@ -138,11 +138,18 @@ function GeoPoint(data) {
     };
   }
 
-  assert(Array.isArray(data) || typeof data === 'object' || typeof data === 'string', 'must provide valid geo-coordinates array [lat, lng] or object or a "lat, lng" string');
+  assert(
+    Array.isArray(data) ||
+    typeof data === 'object' ||
+    typeof data === 'string',
```

Comment:
I think these last two lines should be indented by one extra level, to make it clear they are part of the first argument, as opposed to the next line.

Notice that you have already applied that rule on lines 145 and 146 below.

```js
assert(
  Array.isArray(data) ||
    typeof data === 'object' ||
    typeof data === 'string',
  'must provide valid geo-coordinates array [lat, lng] or object or a ' +
    '"lat, lng" string');
```